### ISSUE TITLE
Set `dropFromOthersDisabled` based on dragged item?

### ISSUE THREAD

**Issue by grantyap** [isaacHagoel/svelte-dnd-action#295]
Is there a way to set `dropFromOthersDisabled` based on the currently dragged item?

For example, I have an item that should not be dropped into a drop zone if a certain field in that item is true. The problem is that `consider()` seems to place the item inside the drop zone first before I can call the function that checks whether that field is true.

Can I make `consider()` just short circuit depending on the currently dragged item?

**Comment by isaacHagoel** [comment]
you can capture the id/state of the dragged item on the first consider event (trigger === TRIGGERS.DRAG_STARTED) and use that to set a the dropFromOthersDisabled where needed. 
If it is not clear, you can make a small REPL that illustrates your usecase and I will assist in filling in the missing part

**Comment by grantyap** [comment]
It seems that the check only happens if I start dragging items from inside that drop zone. Will it work with `TRIGGERS.DRAGGED_ENTERED` instead? Can I return early and just not set `items` if the dragged item shouldn't be added?

After looking at the [multiple types example,](https://svelte.dev/repl/4d23eb3b9e184b90b58f0867010ad258?version=3.24.1) I figured that my use case would be solved if I could use multiple types for one drop zone, i.e. a `gray`-type item can be placed in both `light` and `dark` zones. But it looks like this isn't supported right now.

Thanks for the help by the way, I'll try to get a REPL up later when I have more time.

**Comment by grantyap** [comment]
[Here's the REPL.](https://svelte.dev/repl/acff869a195e4905963a9698cf547b37?version=3.38.2) I'm following the board example.

Here I created the Card and Folder components:

- The Card is just the regular `<div class="card">` turned into a component for reusability.
- The Folder can contain Cards. It also acts like a Card in that it can be moved between Columns and among the Cards.

What I'd like to do is make it so that a Folder cannot contain another Folder. To achieve this, I want to check if the currently dragged item has the `items` field. If it does, then it is a Folder, and we don't allow it to be dropped into other Folders. If it doesn't, it does not contain any items and is thus a Card. 

With what I have right now, it lets you drop Folders into other Folders, which can sometimes display as a Card or sometimes crash. I've tried checking for both `TRIGGERS.DRAG_STARTED` and `TRIGGERS.DRAGGED_ENTERED`, and they both don't seem to work.

I feel like I haven't properly implemented `on:finalize` for Folder in the REPL, but I hope you get idea.
